### ISSUE TITLE
People page doesn't show anything

### ISSUE THREAD

**Issue by monotok** [photoview/photoview-ios#10]
Hello,

Fantastic that you have created a app; finally a way to load images fast via the phone.

However, the people page just stays blank. 

Running iOS 15.4.1 on iPhone 13 Pro.

**Comment by viktorstrate** [comment]
Do you have many faces? Pagination for people have not been added yet on mobile.

**Comment by monotok** [comment]
Yes there are quite a few as I let it scan all 30k images in my library. 

So do you think it would load eventually if I waited?
